cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -200,8 +200,8 @@ public void execute() {
             // when a single bal file is provided.
             if (this.compile) {
                 CommandUtil.printError(this.errStream,
-                        "'-c' or '--compile' flag cannot be used on ballerina files. the flag can only be used with " +
-                        "ballerina projects.",
+                        "The '-c' or '--compile' flag cannot be used with Ballerina files. The flag can only be used with " +
```

Comment:
Is it correct to refer to Ballerina source files simply as Ballerina files?

Reply:
It is implied that it is a source file.